pkg/sdk/action.go

```diff
@@ -54,6 +56,43 @@ func Create(object Object) (err error) {
 	return nil
 }
 
+// Patches provided object on the server with given patch and updates the arg
+// "object" with the result from the server(UID, resourceVersion, etc).
+// Returns an error if the object’s TypeMeta(Kind, APIVersion) or ObjectMeta(Name, Namespace) is missing or incorrect.
+// Returns an error if patch couldn't be json serialized into bytes.
+// Can also return an api error from the server
+// e.g Conflict https://github.com/kubernetes/apimachinery/blob/master/pkg/api/errors/errors.go#L428
+func Patch(patch interface{}, object Object, pt types.PatchType) (err error) {
+	name, namespace, err := k8sutil.GetNameAndNamespace(object)
+	if err != nil {
+		return err
+	}
+	gvk := object.GetObjectKind().GroupVersionKind()
+	apiVersion, kind := gvk.ToAPIVersionAndKind()
+
```

Comment:
no new line here

Reply:
you mean remove this empty line ?

again, I copied same structure here as in other actions e.g. Create.

Reply:
yes, we want the variable to be closed to that of its usage. it is just a style issue. maybe we forgot to do that in Create.

Reply:
same as https://github.com/operator-framework/operator-sdk/pull/389#discussion_r209400841